### DISCUSSION TITLE
Console access

### DISCUSSION THREAD

**Discussion by minigranis** [haugene/docker-transmission-openvpn#2463] [General]
I am having problems getting console access to the container.
I recall this has worked in the past but I cant get it to work - I need to do some tweaks (ie send mail after a torrent is complete), any idea as to why I cant get it to work (I am using Portainer as docker "manager").

Brg

**Comment by pkishino** [comment]
Is the Container up and running properly? No console issues here

**Reply by minigranis** [reply]
Container works as it should, but Portainer just flashes the shell for a short while and then exits.
I can choose the shell but neither /bin/bash or bin/ash works (user root).

**Reply by ilike2burnthing** [reply]
Does this occur for any other containers?

Are you using a reverse proxy?
